weather.py

```diff
@@ -68,7 +121,41 @@ def find_min(weather_data):
     Returns:
         The minium value and it's position in the list.
     """
-    pass
+    if weather_data == []:
+        return ()
+
+    min = float(weather_data[0])# minimum value so far in the list 
+    min_i = 0
+  
+    for i in range(1,len(weather_data)):
+
+          if float(weather_data[i]) <= min:
+            min = float(weather_data[i]) 
+            min_i = (i)
```

Comment:
Great implementation and clean way to manage the initial value of `min`. Be careful of indentation here, some lines are 4-spaces indented and some are 2-spaces indented - may cause some compilation issues or make the code a bit harder to read.